yarn.lock

```diff
@@ -9062,21 +9062,11 @@ header-case@^1.0.0:
     no-case "^2.2.0"
     upper-case "^1.1.3"
 
-hermes-engine@^0.2.1:
-  version "0.2.1"
-  resolved "https://registry.yarnpkg.com/hermes-engine/-/hermes-engine-0.2.1.tgz#25c0f1ff852512a92cb5c5cc47cf967e1e722ea2"
-  integrity sha512-eNHUQHuadDMJARpaqvlCZoK/Nitpj6oywq3vQ3wCwEsww5morX34mW5PmKWQTO7aU0ck0hgulxR+EVDlXygGxQ==
-
```

Comment:
Do we want to be using hermes for macOS? The test app supports that with the latest version I believe

Reply:
No leave it disabled. There's a lot of APEX pushback against Hermes right now. Until we have clear data on how it's better than JavascriptCore, we shouldn't use it. Even then we might be better off just asking Apple to fix whatever deficiencies JSC has instead of using Hermes

Reply:
we don't want to increase app size right now

Reply:
OK, There was no change to enable hermes, so we should still not be using it